webpack.config.js

```diff
@@ -55,4 +57,13 @@ module.exports = {
         new webpack.optimize.UglifyJsPlugin(),
         new webpack.optimize.DedupePlugin(),
     ],
+    module: {
+        loaders: [
+            {
+                test: /\.js$/,
+                exclude: /(node_modules|bower_components|javascripts-legacy|components|vendor)/,
```

Comment:
yeah it seems to run all deps of modules through the loader if you only specify includes. 

given:

```js
test: /\.js$/,
include: [
    path.resolve(__dirname, 'static', 'src', 'javascripts'),
],
loader: 'babel-loader',
```
you get this:
```js
// static/src/javascripts-legacy/projects/common/utils/raven.js 
// (dep of boot-webpack.js)
const guardian = window.guardian;
const x = "ewwerwe"
```
becomes
```js
// static/target/javascripts/boot-webpack.js
var guardian = window.guardian;
var x = "ewwerwe";
```